webpack.config.js

```diff
@@ -19,8 +19,20 @@ module.exports = {
         }
       },
       {
-        test: /\.css$/i,
+        test: /node_modules\/.+\.css$/i,
         use: ["style-loader", "css-loader"]
```

Comment:
why is node_modules included here?

Reply:
This project uses @reach/tabs which requires that you load its css file as global css. So we want css in the node modules to be global but css in the source tab to be css modules. We have two webpack rules now - one for each situation